### ISSUE TITLE
are UnoCSS and Dark mode really necessary??

### ISSUE THREAD

**Issue by vixalien** [ije/html#4]
Hello!

So I noticed that there are many things now in `html` that are kinda undocumented. I'm sorry I'm creating an issue but you didn't enable Discussions so...

First there is `unocss`. I hoped that all the package would do is convert JSX to html without other things. It may be a good thing to disable `unocss` by default or perhaps adding a middleware system or perhaps users could do it themselves if they want `unocss`. I don't really believe `unocss` is necessary.

Secondly, there is the dark theme thing. It's a good idea to provide the `dark` param in the config, but it isn't a good idea to run a script that autodetects the current theme and set it. I think that should be opt in too.

In short, here is what I meant:

- Remove UnoCSS or provide option to enable/disable it.
- Remove the scripts that checks/sets localstorage about dark mode or give option to disable it.

Thanks.

**Comment by ije** [comment]
i agree with you!

**Comment by ije** [comment]
added the unocss as plugin in **0.9.0**:

```jsx
import { html } from "https://deno.land/x/htm/mod.tsx";
import { UnoCSS } from "https://deno.land/x/htm/plugins.tsx";

html.use(UnoCSS())
```

**Comment by vixalien** [comment]
Yay!!
